.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,36 @@
+name: PR build
```

Comment:
Lets add the dependency upgrade workflow too

Reply:
Added in 13d7151